source/auth/auth.md

```diff
@@ -1203,9 +1203,12 @@ in the MONGODB-OIDC specification, including sections or blocks that specificall
 
 > [!NOTE]
 > Drivers MUST NOT url-decode the entire `authMechanismProperties` given in an connection string when the
-> `authMechanism` is `MONGODB-OIDC`. This is because the `TOKEN_RESOURCE` itself will typically be a URL containing a
-> `:` character. The values of the individual `authMechanismProperties` MUST still be url-decoded when given as part of
-> the connection string, and MUST NOT be url-decoded when given as part of a `MongoClient` property.
+> `authMechanism` is `MONGODB-OIDC`. This is because the `TOKEN_RESOURCE` itself will typically be a URL and may contain
+> a `,` character. The values of the individual `authMechanismProperties` MUST still be url-decoded when given as part
+> of the connection string, and MUST NOT be url-decoded when not given as part of the connection string, such as through
+> a `MongoClient` or `Credential` property. Drivers MUST parse the `TOKEN_RESOURCE` by splitting only on the first `:`
+> character. Drivers MUST document that users must url-encode `TOKEN_RESOURCE` when it is provided in the connection
+> string and it contains and of the special characters in \[`,`, `+`, `&`\].
```

Comment:
I think this list must include `%` and might also include the chars `;?`(perhaps others?).
```suggestion
> string and it contains and of the special characters in \[`,`, `+`, `&`, `%`\].
```

(I read this requirement as providing a minimal list, so drivers can document "always encode", and it would meet this requirement.)